financial-connections/src/main/java/com/stripe/android/financialconnections/features/accountpicker/AccountPickerScreen.kt

```diff
@@ -291,7 +289,6 @@ private fun MultiSelectContent(
             AccountItem(
                 selected = allAccountsSelected,
                 onAccountClicked = { onSelectAllAccountsClicked() },
-                enabled = true,
```

Comment:
enabled and allowSelection were redundant. Removed enabled from the model and always rely on the allowSelection field in the account model.